genpipes/tools/wizard.py

```diff
@@ -382,29 +412,54 @@ def valid_step_range(self):
+                    self.add_space()
                     logger.error(f"ERROR!'{part}' not in the correct step range format.")
+                    self.add_space()
                     return False
                 if start > end or start < valid_start or end > valid_end:
+                    self.add_space()
                     logger.error(f"ERROR! Range '{part}' is out of bounds.\nPlease enter a valid step range within these bounds: ({valid_start}-{valid_end}).")
+                    self.add_space()
                     return False
             else:
                 try:
                     step = int(part)
                 except ValueError:
+                    self.add_space()
                     logger.error(f"ERROR! '{part}' is not a number.")
+                    self.add_space()
                     return False
                 if step < valid_start or step > valid_end:
+                    self.add_space()
                     logger.error(f"ERROR! Step '{step}' is out of bounds.\nPlease enter a valid step range within these bounds: ({valid_start}-{valid_end}).")
+                    self.add_space()
                     return False
 
         return True
 
 #for testing
 def main():
-    logger.info("\nWelcome to the GenPipes Wizard!")
-    logger.info("This tool will help you select the appropriate deployment method, pipeline, protocol, and/or construct the command to run GenPipes.")
-    logger.info("Press Ctrl+C at any time to exit the wizard.")
-    logger.info("Let's begin!\n")
+    logger.info(r"""
+    __        __   _                            _          _   _                                               .-+: 
+    \ \      / /__| | ___ ___  _ __ ___   ___  | |_ ___   | |_| |__   ___                                   .-+*--*- 
+     \ \ /\ / / _ \ |/ __/ _ \| '_ ` _ \ / _ \ | __/ _ \  | __| '_ \ / _ \                                 :+#=.   =*:
+      \ V  V /  __/ | (_| (_) | | | | | |  __/ | || (_) | | |_| | | |  __/                               :=#+.   ... -*:
+      _\_/\_/ \___|_|\___\___/|_| |_| |_|\___| _\__\___/ __\__|_| |_|\___|    _ _                     .-#*.      :*=. =*: 
+     / ___| ___ _ __ |  _ \(_)_ __   ___  ___  \ \      / (_)______ _ _ __ __| | |                 .-#*.       .=#+=#*=**:
+    | |  _ / _ \ '_ \| |_) | | '_ \ / _ \/ __|  \ \ /\ / /| |_  / _` | '__/ _` | |              .:##:        .=#=     :*%+. 
+    | |_| |  __/ | | |  __/| | |_) |  __/\__ \   \ V  V / | |/ / (_| | | | (_| |_|              ++.          =%- 
+     \____|\___|_| |_|_|   |_| .__/ \___||___/    \_/\_/  |_/___\__,_|_|  \__,_(_)              .#-         .+=   
+                             |_|                                                                -*:            -#.
+                                                                                                :#*%@%=:  :+%@@*.-#:
+    The GenPipes Wizard will help you select the appropriate deployment method,                  *+.            .++
```

Comment:
Pls replace 'the appropriate' to `an appropriate`.  Use is selecting 'an appropriate' method via the Wizard. Wizard is not enforcing 'the appropriate' method.